src/styles/utils/variables.scss

```diff
@@ -0,0 +1,15 @@
+$dark-blue: #060b35;
+$grey: #616070;
+$light-blue: #00acdc;
+$white: #fff;
+$light-grey:#f3f3f3;
+$font-size-small: 10px;
+$font-size-regular: 12px;
+$font-size-big: 14px;
+$font-size-biggest: 16px;
+$font-weight-regular: 400;
+$font-weight-medium: 500;
+$font-weight-bold: 700;
+$card-width: 200px;
+$card-gapX: 46px;
+$card-gapY: 50px;
```

Comment:
try to use gaps only for spacing cards from each other, - use `Y: 48px`
and for spacing first cards row from nav menu - just apply padding-top on the cards container
![Snapshot 2022-03-31 23 08 19](https://user-images.githubusercontent.com/39308717/161141198-8b94c836-28ab-44f6-a656-3cc33b5972de.png)